python/triton/language/core.py

```diff
@@ -710,6 +710,28 @@ def zeros(shape, dtype, _builder=None):
     return semantic.zeros(shape, dtype, _builder)
 
 
+@builtin
+def full(shape, value, dtype, _builder=None):
```

Comment:
Can we reuse this code path for `tl.zeros` ?

Reply:
Yeah, I believe we could. I'll make changes